ads/_config.js

```diff
@@ -523,6 +523,12 @@ const adConfig = jsonConfiguration({
     renderStartImplemented: true,
   },
 
+  'ssp': {
```

Comment:
Nit: these configs are sorted in alphabetical order. Pls move it to match the file.

Reply:
Sorted alphabeticaly.